gradle.properties

```
@@ -3,14 +3,14 @@ mod_id = authme
 mod_name = Auth Me
 mod_description = Authenticate yourself in Minecraft and re-validate your session
 ## {x-release-please-start-version}
-mod_version = 8.0.0+1.20.4
+mod_version = 8.0.0+1.20.5
```

```suggestion
mod_version = 8.0.0+1.20.4
```
We'll let the CI pipeline update the version. :)